src/utils/JobLogUtils.py

```diff
@@ -0,0 +1,38 @@
+from elasticsearch import Elasticsearch
+import logging
+
+logger = logging.getLogger(__name__)
+
+from config import config
+
+def GetJobLog(jobId, cursor=None, size=None):
+    try:
+        elasticsearch = Elasticsearch(
+            config['elasticsearch'],
+            sniff_on_start=True,
+            sniff_on_connection_fail=True,
+        )
+        request_json = {
+            "query": {
+                "match_phrase": {
+                    "kubernetes.labels.jobId": jobId
+                }
+            },
+            "sort": ["@timestamp"],
+            "_source": ["log", "kubernetes.pod_name", "docker.container_id"]
+        }
+        if cursor is not None:
+            request_json['search_after'] = [cursor]
+        if size is not None:
+            request_json['size'] = size
+        response_json = elasticsearch.search(index="logstash-*", body=request_json)
+        documents = response_json["hits"]["hits"]
+
+        next_cursor = None
+        if len(documents) > 0:
+            next_cursor = documents[-1]["sort"][0]
```

Comment:
Does ` documents[-1]["sort"][0]` mean that each document has one line?

Reply:
It's referring to the first member in line 21 of the file:

```
"sort": ["@timestamp"],
```

Which means the cursor is based on `@timestamp`